src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioDocumentTracker.cs

```diff
@@ -216,51 +203,6 @@ public void Unsubscribe()
             OnContextChanged(kind: ContextChangeKind.ProjectChanged);
         }
 
-        private void StartComputingTagHelpers()
```

Comment:
This is a really nice improvement for this part of the code and it can probably be separated from this PR.